airflow/www/views.py

```diff
@@ -2782,10 +2803,14 @@ def autocomplete(self, session=None):
             ~DagModel.is_subdag, DagModel.is_active,
             DagModel.owners.ilike('%' + query + '%'))
 
-        # Hide paused dags
-        if request.args.get('showPaused', 'True').lower() == 'false':
+        # Hide DAGs if not showing status: "all"
```

Comment:
Not sure off the top of my head how this auto-complete works, but might it be better to allow auto-completing of all dags, even ones that aren't shown? Or is the filtering/searching done client side based on the visible data only?

Reply:
I guess this is the "same" behaviour as before, just updated to the new mechanism so this is fine.